src/docfx/build/metadata/ContributionInfo.cs

```diff
@@ -122,7 +122,7 @@ public DateTime GetUpdatedAt(Document document, List<GitCommit> fileCommits)
                 return default;
 
             var branch = repo.Branch ?? "master";
-            var editRepo = document.Docset.Config.Contribution.Repository ?? $"{repo.Owner}/{repo.Name}";
+            var editRepo = LocConfigConversion.GetEditRepository(document.Docset.Config.Contribution.Repository, document.Docset.Locale) ?? $"{repo.Owner}/{repo.Name}";
```

Comment:
naming consistent, shall we use loc or locale?

Reply:
should be convention?

Reply:
loc means localization, not locale